package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "magic-calculator",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.15.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
+  },
+  "scripts": {
+    "start": "react-scripts --openssl-legacy-provider start",
+    "build": "react-scripts --openssl-legacy-provider build",
```

Comment:
- [ ] The script is not configured correctly, I can't start the app.

Reply:
@dasileker Thank you for your review.
I use this `--openssl-legacy-provider` for both `build` and `start` as node v17.0.1 which I use, do not support `digital envelope routines` 
I have recorded a video which shows the working of the app
https://www.loom.com/share/57ded24937fe45cd8e9447270cc77259

Reply:
The meaning of making it working globally is that not everyone has the same node version installed.

So keep in mind that someone need to run the app to see the project, but hz can't because it's configured to be used just for one version it will be not good.

Reply:
@dasileker. Thank you for your response. I found a workaround by running this on my system `export NODE_OPTIONS=--openssl-legacy-provider` 
Now I don't have to use the additional parameter `--openssl-legacy-provider` 
is it possible for you to once verify it now running on your system?